src/init/omp.bash

```diff
@@ -4,23 +4,27 @@ TIMER_START="/tmp/${USER}.start.$$"
 
 PS0='$(::OMP:: --millis > $TIMER_START)'
 
-function _update_ps1() {
+function _omp_hook() {
+    local ret=$?
+
     omp_elapsed=-1
     if [[ -f $TIMER_START ]]; then
         omp_now=$(::OMP:: --millis)
         omp_start_time=$(cat "$TIMER_START")
-        omp_elapsed=$(($omp_now-$omp_start_time))
-        rm $TIMER_START
+        omp_elapsed=$((omp_now-omp_start_time))
+        rm "$TIMER_START"
     fi
-    PS1="$(::OMP:: --config $POSH_THEME --error $? --execution-time $omp_elapsed --shell bash)"
+    PS1="$(::OMP:: --config $POSH_THEME --error $ret --execution-time $omp_elapsed)"
```

Comment:
the `--shell bash` statement would need to remain as it causes issues running this on Apple silicon otherwise.

Reply:
oh damn I lost that piece while copypasting the command between the shell and the file. I'll fix it asap